tests/openedx_tagging/core/tagging/test_system_defined_models.py

```diff
@@ -205,29 +205,33 @@ class TestLanguageTaxonomy(TestTagTaxonomyMixin, TestCase):
     Test for Language taxonomy
     """
 
-    @ddt.data(
-        ("en", "tag_id"),  # Valid
-        ("es", "tag_id"),  # Not available lang
-        ("en", None),  # Test parent validations
-    )
-    @ddt.unpack
-    def test_validations(self, lang, tag_id):
-        tag = Tag(
-            id=tag_id,
-            taxonomy=self.language_taxonomy,
-            value="_val",
-            external_id=lang,
-        )
-        object_tag = ObjectTag(
-            object_id="id",
-            tag=tag,
-        )
-        self.language_taxonomy.validate_object_tag(
```

Comment:
This function just returns a `bool` so this test was having no effect. I commented it out because again I'm not sure what it's supposed to be doing.

Reply:
@ChrisChV Same here.

Reply:
Ack!